samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/main.c

```diff
@@ -185,6 +187,10 @@ void main(void)
 
 	app_gpio_init();
 
+#ifdef ENABLE_SMP_SVR
```

Comment:
I think this should probably be done using `SYS_INIT()` at application priority in src/smp_svr.c instead. That way it is done if and only if necessary and the main file doesn't need the ifdef.